bin/crew

```diff
@@ -311,9 +311,10 @@ def update
   # update compatible packages
   generate_compatible
   # check for outdated installed packages
-  puts 'Checking for package updates...'
+  puts 'Checking for package updates...' unless CREW_UNATTENDED
```

Comment:
What does it hurt to print this message anyway?

Reply:
The goal is not to have to strip it out afterwards... It simplifies the logic externally.